src/utils/calldata/cairo.ts

```diff
@@ -37,6 +37,10 @@ export const getArrayType = (type: string) => {
 export function isCairo1Abi(abi: Abi): boolean {
   const firstFunction = abi.find((entry) => entry.type === 'function');
   if (!firstFunction) {
+    if (abi.find((it) => it.type === 'interface')) {
```

Comment:
This entire codebase needs the next refactoring as so that this is part of the parser or factory parser, but this is just as is, as the full solution would take week/s to do.